README.md

```diff
@@ -62,6 +63,25 @@ ethminer --help
 
 Check our [samples](POOLEXSAMPLES_ETH.md) to see how to connect to different pools.
 
+### Examples connecting proxy
+
+Ethminer supports using a proxy server, such as eth-proxy: https://github.com/Atrides/eth-proxy
+
+If multiple miners are running ethminer they can all connect to a single proxy. This increases the efficiency of the mining operation. 
```

Comment:
The statement "This increases efficiency of the mining operation" is false and misleading.